### PR TITLE
Update /setspawnworld

### PR DESCRIPTION
**Pull request by DUFCLiam**
Add this [-b] as this will broadcast the "[SERVER] DUFC_Liam: New spawn!"

### CODE REVIEW

**Comment by DUFCLiam** [comment]
Forgot issue

**Comment by DUFCLiam** [comment]
Wait..... @JeromSar Can you add the "PR: On hold" label here please.

File: src/main/java/me/totalfreedom/totalfreedommod/commands/Command_setspawnworld.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = PlayerRank.SUPER_ADMIN, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Set world spawnpoint.", usage = "/<command>")
+@CommandParameters(description = "Set world spawnpoint.", usage = "/<command> [-b]")
 public class Command_setspawnworld extends FreedomCommand
```
Comment (rsajeey) [inline src/main/java/me/totalfreedom/totalfreedommod/commands/Command_setspawnworld.java]: Should be "TFM_Command" instead of "FreedomCommand".

**Comment by DUFCLiam** [comment]
This is because I got it from TFM 5.0

**Comment by DUFCLiam** [comment]
Thanks Jerom

**Comment by LegendIsAwesomes** [comment]
This would not work,  you've mixed TFM 5.0 features and mixed them with the current TFM.

**Comment by JeromSar** [comment]
The associated issue has been closed.

**Pull request closed without merging**